### PR TITLE
```CNDIT-1045`` :  Update the Provider service to integrate with the Debezium Change data events

### PR DESCRIPTION
**Pull request by alog-enquizit**
Update the `sp_provider_event` stored proc to remote the jobflow log dependencies and the transaction parts.

Update the Provider service to integrate with the debezium change data. Update the Provider service to be process the data from the `cdc.nbs_odse.dbo.Person` topic

### CODE REVIEW

File: etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/controller/DataPipelineController.java [hunk 1]
```diff
@@ -37,7 +41,7 @@ public ResponseEntity<String> getDataPipelineStatusHealth() {
     public ResponseEntity<String> postProvider(@RequestBody String payLoad) throws JsonProcessingException {
         KafkaProducer<String, JsonNode> producer = new KafkaProducer<>(
                 kafkaStreamsConfig.kStreamsConfigs().asProperties());
-        producer.send(new ProducerRecord<>("cdc.nbs_odse.dbo.Provider",
+        producer.send(new ProducerRecord<>(kafkaConfig.getPersonTopicName(),
```
Comment (upasanapattnaik-eq) [inline etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/controller/DataPipelineController.java:44]: We would need to create this function for each topic, correct?

File: etl-data-pipeline/src/main/resources/db/routines/provider/sp_provider.sql [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 Create or Alter PROCEDURE [dbo].[sp_PROVIDER_EVENT] @user_id_list varchar(max)
 as
 BEGIN
 
     --UPDATE ACTIVITY_LOG_DETAIL SET
---START_DATE=DATETIME();
+    --START_DATE=DATETIME();
 
-    DECLARE @RowCount_no INT;
     DECLARE @Proc_Step_no FLOAT = 0;
```
Comment (upasanapattnaik-eq) [inline etl-data-pipeline/src/main/resources/db/routines/provider/sp_provider.sql:10]: We are removing these steps as well?

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM

File: etl-data-pipeline/src/main/resources/db/routines/provider/sp_provider.sql (hunk 2 above)
Comment (alog-enquizit) [reply etl-data-pipeline/src/main/resources/db/routines/provider/sp_provider.sql:10]: Yeah we want to keep the stored proc simple. Let me remove these too.

Comment (alog-enquizit) [reply etl-data-pipeline/src/main/resources/db/routines/provider/sp_provider.sql:10]: I will keep them for no for documentation purposes as to modularize the stored proc into different steps. They are just harmless titles on top of each step. If we decide to move away from them, we can remove these and morph into comments.

File: etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/controller/DataPipelineController.java (hunk 1 above)
Comment (alog-enquizit) [reply etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/controller/DataPipelineController.java:44]: Yeah but This is just for testing purposes. The actual data event should be sent by Debezium Change Data Capture module.

**Comment by alog-enquizit** [comment]
> LGTM

Thanks for reviewing!

**Pull request merged**